sanic.py

```diff
@@ -1,19 +1,24 @@
 from discord.ext import commands
-
-from sanicbot.core.cogs import HelpCog, GitCog
+from discord import Embed
 from sanicbot.core.config import config
+import aiohttp
+from asyncio import run as asynciorun
 
-bot = commands.Bot(help_command=None, command_prefix=None)
-git_cog = GitCog(bot)
+class embeddedHelpCommand(commands.MinimalHelpCommand):   # help command embedded, subclassed of the minimal help command
+    async def send_pages(self):
+        destination = self.get_destination()    #get where to send
+        with open("./resources/help.txt") as f:
+            embed = Embed(description=f.read()) # set desc as page  
+        await destination.send(embed=embed) # send embed
 
 
-@bot.event
-async def on_message(message):
-    await git_cog.github_issue_message_listener(message)
+async def startup():
```

Comment:
This startup method becomes obsolete by extending the `commands.Bot` class. (see PR [#8] for implementation) 
This class also: 
- adds `bot.httpclient` at the bot level so it can be accessed in the same way as `bot.session`.
- adds all `REGISTERED_COGS` with `self.add_cog(cog(self))`
- loads slash commands using `self.add_all_cog_commands()`
- attaches a bot level logger which can be used as such: `bot.logger.info(...)`
- overrides events such as `on_connect` and `on_ready` at the bot level
- overrides `on_close` to gracefully close `httpclient`